css/media.css

```diff
@@ -0,0 +1,1039 @@
+    font-size: 12px;
+  }
+
+  .banner__link-more::after {
+    margin-left: 8px;
+    content: url(../img/Arrow.svg);
+  }
+
+  .section__subtitle {
+    font-size: 20px;
+  }
+
+  .market-section__subtitle {
+    margin: 34px 12px 0 0;
+  }
+
+  .product-card {
+    padding: 0 0 0 14px;
+  }
+
+  .products__product-card:nth-last-child(-n+3) {
+    display: none;
+  }
+
+  .product-card__picture {
+    margin: 0 0 24px 45px;
+  }
+
+  .product-card__picture img {
+    width: 166px;
+    height: 152px;
+  }
+
+  .product-card__name {
+    font-size: 15px;
+    margin: 0 0 4px 0;
+  }
+
+  .product-card__price {
+    font-size: 20px;
+  }
+
+  .circles {
+    margin: 4px 0 0 57px;
+  }
+
+  .circles__link {
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
+  }
+
+  /* promo-section */
+  .promo-section {
+    position: relative;
+    padding-bottom: 84px;
+  }
+
+  .promo-section::before {
+    top: -20px;
+    height: 30px;
+  }
+
+  .video-block {
+    min-height: 450px;
+  }
+
+  .video-block__text-wrapper {
+    padding: 0;
+  }
+
+  .video-block__slogan {
+    display: inline-block;
+    max-width: 250px;
+    margin: 0 auto;
+    margin-top: 16px;
+    font-size: 20px;
+    line-height: 23px;
+  }
+  
+  .wide {
+    margin-right: 6px;
+  }
+
+  .video-block__text {
+    display: inline-block;
+    max-width: 75%;
+    margin: 18px 0 27px 0;
+    line-height: 18px;
+    font-size: 14px;
+  }
+
+  .video-block__picture-wrapper {
+    margin: 0;
+  }
+
+  .video-block__pic {
+    width: 284px;
+  }
+
+  .promo-section__subtitle {
+    margin-top: 0;
+    margin-bottom: 27px;
+  }
+
+  .stories-card {
+    max-width: 280px;
+    margin: 0 auto;
+  }
+
+  .stories-card__pic {
+    margin-left: 1px;
+    min-height: 148px;
+  }
+
+  .stories-card__title {
+    font-size: 20px;
+    margin: 27px 0 8px 1px;
+  }
+
+  .stories-card__descr {
+    margin-bottom: 20px;
+  }
+
+  .gallery {
+    max-height: 420px;
+    max-width: 280px;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .gallery__hashtag {
+    margin: 98px 0 10px 7px;
+    font-size: 25px;
+    text-align: center;
+  }
+
+  .gallery__photo {
+    width: 50%;
+    border-radius: 0;
+  }
+
+  .gallery__photo:last-child {
+    display: none;
+  }
+
+  .gallery__photo:nth-child(4) {
+    display: none;
+  }
+
+  .gallery__photo:nth-child(3) {
+    order: 6;
+  }
+
+  .gallery__photo:nth-child(5) {
+    order: 3;
+  }
+  
+  .promo-section::after {
+    height: 289px;
+  }
+}
+
+@media screen and (max-width: 320px) {
+  .container {
+    max-width: 320px;
+  }
+  
+  .header {
+    padding: 0 4px;
+  }
+
+  .header__logo-mobile {
+    margin-top: 19px;
+  }
+
+  .burger {
+    height: 5px;
+    margin-top: 7px;
+  }
+
+  .banner {
+    padding: 50px 0 0 20px;
+    background-position: 24%;
+  }
+
+  .video-block {
+    padding: 0 18px;
+  }
+
+  .video-block__picture-wrapper {
+    margin: 0;
+  }
+
+  .video-block__text {
+    max-width: 99%;
+  }  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654145)